packages/components/field/src/js/field.controller.js

```diff
@@ -149,7 +149,7 @@ export default class FieldController {
     }
 
     this.checkAllErrors();
-    return this.invalid && !this.hasFocus && (this.blurred || this.form.$submitted);
+    return this.invalid && (this.blurred || this.form.$submitted);
```

Comment:
Why not cleaning all `this.hasFocus` or functions using it  in controllers, if it has no other purpose now ?
I see occurrences in `field`, `select` and `dropdown` (really unused for the last one).

I see too that we call `this.checkControlErrors(controlElement, name);` only on blur event. 
Is that what we want with this change ?

Reply:
I have cleaned the occurences of `hasFocus`.
I don't understand the purpose of `checkControlErrors` so I let it there :-)